packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -1508,6 +1513,7 @@ class SelectionOverlay {
   /// {@endtemplate}
   void dispose() {
     hide();
+    _magnifierInfo.dispose();
```

Comment:
I didn't realize we had to `.remove` and `.dispose` these things, I'll keep an eye out for that in the future.